pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -90,7 +90,7 @@ type PreEnqueueCheck func(pod *v1.Pod) bool
 type SchedulingQueue interface {
 	framework.PodNominator
 	Add(pod *v1.Pod) error
-	// Activate moves the given pods to activeQ iff they're in unschedulablePods or backoffQ.
+	// Activate moves the given pods to activeQ if they're in unschedulablePods or backoffQ.
```

Comment:
It's correct because `iff` means `if and only if`.
TBH, I sent the same PR as you in the past 😅 
https://github.com/kubernetes/kubernetes/pull/104542